client/src/App.js

```diff
@@ -17,31 +18,20 @@ function App() {
     of a JSX element will be read as text like HTML and not as JavaScript. */
   return (
     <div className="App">
-
       {/* <header className="App-header"> */}
 
       {/* Should we be including all this text in header? Should we also be using body? */}
-      <header className="App-header"> 
-        <nav className= "nav-bar">
-          <p>{"Sign Up"}</p>
+      <header className="App-header">
+        <nav className="nav-bar">
+          <Button text="Sign Up" />
         </nav>
-        <section className="hero">
-        
-        </section>
-        <section className="app-info">
-
-        </section>
-        <section className="your-pick">
-
-        </section>
-        <section  className="palette">
-
-        </section>
-        <footer className="bottom">
-
-        </footer>
+        <section className="hero"></section>
+        <section className="app-info"></section>
+        <section className="your-pick"></section>
+        <section className="palette"></section>
+        <footer className="bottom"></footer>
```

Comment:
Fantastic, thanks for the cleanup

Reply:
Installing Prettier helps a lot. It helps to format code automatically for you.